src/dune_rules/module_compilation.ml

```diff
@@ -196,49 +211,45 @@ let build_cm cctx ~force_write_cmi ~precompiled_cmi ~cm_kind (m : Module.t)
   in
   let src =
     match phase with
-    | Some Emit -> Path.build linear_fdo
+    | Some Emit ->
+      let linear_fdo =
```

Comment:
Why not inline this?

Reply:
Presumably the original author of this code made this name on purpose. I'm totally unfamiliar with fdo so I'm not sure if it helps readability or not.